### PR TITLE
Add message alert, license check and warning msg to CBT

### PR DESCRIPTION
**Pull request by jijiang**

### CODE REVIEW

File: XenModel/FriendlyNames.resx [hunk 1]
```diff
@@ -1076,6 +1076,15 @@
   <data name="Message.body-vcpu_qos_failed" xml:space="preserve">
     <value>Quality of Service settings for VCPU '{0}' on virtual machine '{1}' could not be obeyed.</value>
   </data>
+  <data name="Message.body-vdi_cbt_metadata_inconsistent" xml:space="preserve">
```
Comment (MihaelaStoica) [inline XenModel/FriendlyNames.resx]: We need better messages here, both for the body and for the name.

Comment (thomasmck) [reply XenModel/FriendlyNames.resx]: Below are my suggestions for the name and body of the messages. Note that I have used 2 different sets of instructions in the body (italicized) as I wasn't sure if the first was clear enough. Please do not take this as definitive so feel free to change them if you think the wording is dodgy or unclear.

**Inconsistent**
Name: Changed block tracking disabled due to inconsistent metadata.
Body: Changed block tracking disabled for VDI {0} due to inconsistent metadata. _To continue using changed block tracking with VDI {0} re-enable changed block tracking, take a snapshot and back it up._

**Resize**
Name: Changed block tracking disabled due to VDI resizing failure.
Body: Changed block tracking disabled for VDI {0} due to VDI resizing failed. _To continue using changed block tracking with VDI {0} re-enable changed block tracking, take a snapshot and back it up._

**Snapshot**
Name: Changed block tracking disabled due to snapshot failure.
Body: Changed block tracking disabled for VDI {0} due to snapshot failure. _To continue using changed block tracking with VDI {0} you will need to re-do the whole backup setup including re-enabling changed block tracking, taking a snapshot and backing it up._

File: XenModel/FriendlyNames.resx [hunk 2]
```diff
@@ -1277,6 +1286,15 @@
   <data name="Message.name-vcpu_qos_failed" xml:space="preserve">
     <value>VM vCPU QoS Failed</value>
   </data>
+  <data name="Message.name-vdi_cbt_metadata_inconsistent" xml:space="preserve">
+    <value>Changed block tracking disabled due to metadata inconsistent</value>
```
Comment (GaborApatiNagy) [inline XenModel/FriendlyNames.resx]: I think it should be either "metadata inconsistency" or "the metadata being inconsistent". I prefer the first though.

Comment (thomasmck) [reply XenModel/FriendlyNames.resx]: My suggestion is `Changed block tracking disabled due to inconsistent metadata`

Comment (GaborApatiNagy) [reply XenModel/FriendlyNames.resx]: Cool. I'd prefer @thomasmck 's version. Let's take that.

File: XenModel/Messages.resx [hunk 3]
```diff
@@ -2696,10 +2696,10 @@ This action is final and unrecoverable.</value>
     <value>&amp;Yes, Destroy</value>
   </data>
   <data name="CONFIRM_DISABLE_CBT_VM" xml:space="preserve">
-    <value>This will disable Changed Block Tracking on the disks of the selected VM. Do you want to continue?</value>
+    <value>This will disable Changed Block Tracking on the disks of the selected VM. If you are using any third-party solutions to back up the VM, they might be affected. Note that Changed Block Tracking cannot be enabled again from XenCenter. Do you want to continue?</value>
```
Comment (MihaelaStoica) [inline XenModel/Messages.resx]: Please use [XenCenter]

File: XenModel/FriendlyNames.resx [hunk 4]
```diff
@@ -1076,6 +1076,15 @@
   <data name="Message.body-vcpu_qos_failed" xml:space="preserve">
     <value>Quality of Service settings for VCPU '{0}' on virtual machine '{1}' could not be obeyed.</value>
   </data>
+  <data name="Message.body-vdi_cbt_metadata_inconsistent" xml:space="preserve">
+    <value>Changed block tracking is disabled for virtual disk {0} due to inconsistent metadata. To continue using changed block tracking with the virtual disk, you will need to do the backup setup again, including re-enabling changed block tracking, taking a snapshot and backing it up.</value>
```
Comment (MihaelaStoica) [inline XenModel/FriendlyNames.resx]: ... **has been** disabled...
... with **this** vistual disk .....

... due to **a** resizing failure ...
... due to **a** snapshot failure ...

File: XenModel/FriendlyNames.resx [hunk 5]
```diff
@@ -1277,6 +1286,15 @@
   <data name="Message.name-vcpu_qos_failed" xml:space="preserve">
     <value>VM vCPU QoS Failed</value>
   </data>
+  <data name="Message.name-vdi_cbt_metadata_inconsistent" xml:space="preserve">
+    <value>Changed block tracking disabled due to inconsistent metadata</value>
+  </data>
+  <data name="Message.name-vdi_cbt_resize_failed" xml:space="preserve">
+    <value>Changed block tracking disabled due to resizing failure</value>
```
Comment (MihaelaStoica) [inline XenModel/FriendlyNames.resx]: ... due to **a virtual disk** resizing failure ...
... due to **a** snapshot failure ...

**Pull request merged**